packages/bson-transpilers/test/yaml/native.yaml

```diff
@@ -20,6 +20,10 @@ tests:
           shell: "{\n  'x': '1'\n}"
           object: "{x: '1'}"
           ruby: "{\n  'x' => '1'\n}"
+          swift: |-
```

Comment:
do all of the document and array literals in this file only get used in contexts where the compiler will be able to infer the type?